### PR TITLE
feature Data calendar

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

File: src/main/java/core/utils/DateActualAfterBefore.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        if (sizeArray == 4) {
+            return getDmYsMhFuture(textDate);
+        }
+        return null;
+    }
+
+    public Calendar getDmYsMhFuture(String textDate) {
+        Date actualDate = new Date();
+        String[] arrayTextDate = textDate.split(" ");
+        if (arrayTextDate[1].equals("days")  || arrayTextDate[1].equals("day")) {
```
Comment (SaulCM) [inline src/main/java/core/utils/DateActualAfterBefore.java]: There is an extra space before de OR operator.

File: src/main/java/core/utils/DateActualAfterBefore.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            return addsSubsDays(actualDate, Integer.parseInt(arrayTextDate[0]));
+        }
+        if (arrayTextDate[1].equals("months") || arrayTextDate[1].equals("month")) {
+            return addsSubsMonths(actualDate, Integer.parseInt(arrayTextDate[0]));
+        }
+        if (arrayTextDate[1].equals("years") || arrayTextDate[1].equals("year")) {
+            return addsSubsYears(actualDate, Integer.parseInt(arrayTextDate[0]));
+        }
+        if (arrayTextDate[1].equals("seconds")  || arrayTextDate[1].equals("second")) {
```
Comment (SaulCM) [inline src/main/java/core/utils/DateActualAfterBefore.java]: Extra space before the OR operator.

File: src/main/java/core/utils/DateActualAfterBefore.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        }
+        if (sizeArray == 3) {
+            return getDmYsMhAgo(textDate);
+        }
+        if (sizeArray == 4) {
+            return getDmYsMhFuture(textDate);
+        }
+        return null;
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/core/utils/DateActualAfterBefore.java:35]: Add  javadocs and manager exceptions

Comment (GustavoZHA) [reply src/main/java/core/utils/DateActualAfterBefore.java:35]: like null

File: src/main/java/core/utils/DateActualAfterBefore.java (hunk 1 above)
Comment (SaulCM) [inline src/main/java/core/utils/DateActualAfterBefore.java]: Good practice to put the object to compare first: ("days").arrayTextDate[1]

**Review by SaulCM — APPROVED** [review]

**Pull request merged**